src/main/java/service/DataParserImpl.java

```diff
@@ -0,0 +1,33 @@
+package service;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitDataDto;
+
+public class DataParserImpl implements DataParser {
+
+    public static final String COMMA = ",";
```

Comment:
```suggestion
    public static final String CSV_SEPARATOR = ",";
```
name variables according to their purpose not the value they hold